### PR TITLE
feat(s3): add API calls list - MTA-1874

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

**Comment by angristan** [comment]
Could we use the same S3 action names as https://docs.aws.amazon.com/AmazonS3/latest/API/API_Operations_Amazon_Simple_Storage_Service.html?

File: storage/object/api-cli/multipart-uploads.mdx [hunk 1]
```diff
@@ -310,6 +310,6 @@ $ aws s3api list-multipart-uploads --bucket my-bucket
 ```
 
 <Navigation title="See Also">
-  <PreviousButton to="/storage/object/api-cli/lifecycle-rules-api/">Managing the lifecycle of objects</PreviousButton>
+  <PreviousButton to="/storage/object/api-cli/managing-lifecycle-cliv2">Managing an Object Storage Lifecycle using CLI (v2)</PreviousButton>
```
Comment (RoRoJ) [inline storage/object/api-cli/multipart-uploads.mdx:313]: the Next button points to /storage/object/api-cli/setting-cors-rules/" Setting CORS rules, and that page exists but doesn't have a link on the side (navigation) menu

File: storage/object/api-cli/object-operations.mdx [hunk 2]
```diff
@@ -459,59 +533,39 @@ Authorization: authorization string
...
-Multipart uploads allow you to upload a single object as a set of parts. Each part is a contiguous portion of the object's data. You can upload these object parts independently and in any order. If transmission of any part fails, you can retransmit that part without affecting other parts. After all parts of your object are uploaded, Scaleway Object Storage assembles these parts and creates the object.
-
-<Message type="tip">
-
-Check out our multipart uploads [documentation page](/storage/object/api-cli/multipart-uploads/) for more information.
-
-</Message>
+## Restore Object
 
+This calls moves an object back to the `STANDARD` storage class for the specificed number of days.
```
Comment (RoRoJ) [inline storage/object/api-cli/object-operations.mdx:545]:
```suggestion
This call moves an object back to the `STANDARD` storage class for the specified number of days.
```

File: storage/object/api-cli/using-api-call-list.mdx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+Status:
+  - Feature available: <Icon name="validate" /> 
+  - Feature coming in 2022: ⌛
+
+## Common Operations
+
+| Feature | Description | Status |
+| :------------ | :---- | : - |
+|  [GET Service](/storage/object/api-cli/common-operations/#get-service) | Returns a list of all bucket | <Icon name="validate" /> |
```
Comment (RoRoJ) [inline storage/object/api-cli/using-api-call-list.mdx:29]:
```suggestion
|  [GET Service](/storage/object/api-cli/common-operations/#get-service) | Returns a list of all buckets | <Icon name="validate" /> |
```

File: storage/object/api-cli/using-api-call-list.mdx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+## Bucket Operations
+
+| Feature | Description | Status |
+| :------------ | :---- | : - |
+|  [DELETE bucket](/storage/object/api-cli/bucket-operations/#delete-bucket) |  Deletes bucket | <Icon name="validate" /> |
+|  [DELETE bucket CORS](/storage/object/api-cli/bucket-operations/#delete-bucket-cors) |  Deletes the [CORS](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) configuration of a bucket | <Icon name="validate" /> |
+|  [DELETE bucket lifecycle](/storage/object/api-cli/bucket-operations/#delete-bucket-lifecycle)|  Deletes the [lifecycle configuration](/storage/object/api-cli/lifecycle-rules-api/) of a bucket | <Icon name="validate" /> |
+|  [DELETE bucket tagging](/storage/object/api-cli/bucket-operations/#delete-bucket-tagging) |  Deletes the tag(s) of a bucket | <Icon name="validate" /> |
+|  [DELETE bucket website](/storage/object/api-cli/bucket-operations/#delete-bucket-website) |  This operation deletes the bucket Website configuration of a specified bucket | <Icon name="validate" /> |
```
Comment (RoRoJ) [inline storage/object/api-cli/using-api-call-list.mdx:40]: check capitalization of Bucket Website (can't remember if it's all upper or all lower?) here and in a few other places in this doc

File: storage/object/api-cli/using-api-call-list.mdx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+|  [PUT bucket policy](/storage/object/api-cli/bucket-operations/#put-bucket-policy) |  This operation applies an S3 bucket policy to an S3 bucket. The key elements of bucket policy are [Version](/storage/object/api-cli/bucket-policy/#version), [ID](/storage/object/api-cli/bucket-policy/#id), [Statement](/storage/object/api-cli/bucket-policy/#statement), [Sid](/storage/object/api-cli/bucket-policy/#sid), [Principal](/storage/object/api-cli/bucket-policy/#principal), [Action](/storage/object/api-cli/bucket-policy/#action), [Effect](/storage/object/api-cli/bucket-policy/#effect), [Resource](/storage/object/api-cli/bucket-policy/#resource) and [Condition](/storage/object/api-cli/bucket-policy/#condition). You can find out more about each element by clicking the links, or consulting the full documentation | <Icon name="validate" /> |
+|  [GET bucket policy](/storage/object/api-cli/bucket-operations/#get-bucket-policy) |  This operation returns the policy of a specified bucket | <Icon name="validate" /> |
```
Comment (RoRoJ) [inline storage/object/api-cli/using-api-call-list.mdx:61]:
```suggestion
|  [GET bucket policy](/storage/object/api-cli/bucket-operations/#get-bucket-policy) |  Returns the policy of a specified bucket | <Icon name="validate" /> |
```

File: storage/object/api-cli/using-api-call-list.mdx [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+|  [PUT bucket ACL](/storage/object/api-cli/bucket-operations/#put-bucket-acl) |  Configures the Access Control List of a bucket | <Icon name="validate" /> |
+|  [PUT bucket CORS](/storage/object/api-cli/bucket-operations/#put-bucket-cors) |  Configures [CORS](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) on a bucket | <Icon name="validate" /> |
+|  [PUT bucket lifecycle](/storage/object/api-cli/bucket-operations/#put-bucket-lifecycle)|  Sets the [lifecycle configuration](/storage/object/api-cli/lifecycle-rules-api/) of a bucket | <Icon name="validate" /> |
+|  PUT bucket notification | Enables notifications of specified events for a bucket | ⌛ |
+|  [PUT bucket tagging](/storage/object/api-cli/bucket-operations/#put-bucket-tagging) |  Sets the tag(s) of a bucket | <Icon name="validate" /> |
+|  [PUT bucket versioning](/storage/object/api-cli/bucket-operations/#put-bucket-versioning) |  Sets the versioning state of an existing bucket | <Icon name="validate" /> |
+|  [PUT bucket website](/storage/object/api-cli/bucket-operations/#put-bucket-website) |  This operation enables bucket Website and sets the basic configuration for the website | <Icon name="validate" /> |
```
Comment (RoRoJ) [inline storage/object/api-cli/using-api-call-list.mdx:59]:
```suggestion
|  [PUT bucket website](/storage/object/api-cli/bucket-operations/#put-bucket-website) |  Enables bucket Website and sets the basic configuration for the website | <Icon name="validate" /> |
```

File: storage/object/api-cli/using-api-call-list.mdx [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+|  [PUT bucket policy](/storage/object/api-cli/bucket-operations/#put-bucket-policy) |  This operation applies an S3 bucket policy to an S3 bucket. The key elements of bucket policy are [Version](/storage/object/api-cli/bucket-policy/#version), [ID](/storage/object/api-cli/bucket-policy/#id), [Statement](/storage/object/api-cli/bucket-policy/#statement), [Sid](/storage/object/api-cli/bucket-policy/#sid), [Principal](/storage/object/api-cli/bucket-policy/#principal), [Action](/storage/object/api-cli/bucket-policy/#action), [Effect](/storage/object/api-cli/bucket-policy/#effect), [Resource](/storage/object/api-cli/bucket-policy/#resource) and [Condition](/storage/object/api-cli/bucket-policy/#condition). You can find out more about each element by clicking the links, or consulting the full documentation | <Icon name="validate" /> |
```
Comment (RoRoJ) [inline storage/object/api-cli/using-api-call-list.mdx:60]:
```suggestion
|  [PUT bucket policy](/storage/object/api-cli/bucket-operations/#put-bucket-policy) |  Applies an S3 bucket policy to an S3 bucket. The key elements of bucket policy are [Version](/storage/object/api-cli/bucket-policy/#version), [ID](/storage/object/api-cli/bucket-policy/#id), [Statement](/storage/object/api-cli/bucket-policy/#statement), [Sid](/storage/object/api-cli/bucket-policy/#sid), [Principal](/storage/object/api-cli/bucket-policy/#principal), [Action](/storage/object/api-cli/bucket-policy/#action), [Effect](/storage/object/api-cli/bucket-policy/#effect), [Resource](/storage/object/api-cli/bucket-policy/#resource) and [Condition](/storage/object/api-cli/bucket-policy/#condition). You can find out more about each element by clicking the links, or consulting the full documentation | <Icon name="validate" /> |
```

File: storage/object/api-cli/using-api-call-list.mdx [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+|  [GET bucket policy](/storage/object/api-cli/bucket-operations/#get-bucket-policy) |  This operation returns the policy of a specified bucket | <Icon name="validate" /> |
+|  [DELETE bucket policy](/storage/object/api-cli/bucket-operations/#delete-bucket-policy) |  This operation deletes the policy of a specified bucket | <Icon name="validate" /> |
```
Comment (RoRoJ) [inline storage/object/api-cli/using-api-call-list.mdx:62]:
```suggestion
|  [DELETE bucket policy](/storage/object/api-cli/bucket-operations/#delete-bucket-policy) |  Deletes the policy of a specified bucket | <Icon name="validate" /> |
```

File: storage/object/api-cli/using-api-call-list.mdx [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+|  [GET bucket lifecycle](/storage/object/api-cli/bucket-operations/#get-bucket-lifecycle)|  Returns the [lifecycle configuration](/storage/object/api-cli/lifecycle-rules-api/) of a bucket | <Icon name="validate" /> |
+|  [GET bucket location](/storage/object/api-cli/bucket-operations/#get-bucket-location) |  Returns the region where the bucket is | <Icon name="validate" /> |
+|  GET bucket notification | Returns the notification configuration of a bucket | ⌛ |
+|  [GET bucket object versions](/storage/object/api-cli/bucket-operations/#get-bucket-object-version) |  Returns metadata about all of the versions of objects in a bucket | <Icon name="validate" /> |
+|  [GET bucket tagging](/storage/object/api-cli/bucket-operations/#get-bucket-tagging) |  Returns the tag(s) of a bucket | <Icon name="validate" /> |
+|  [GET bucket versioning](/storage/object/api-cli/bucket-operations/#get-bucket-versioning) |  Returns the versioning state of a bucket | <Icon name="validate" /> |
+|  [GET bucket website](/storage/object/api-cli/bucket-operations/#get-bucket-website) |  This operation returns the bucket Website basic configuration | <Icon name="validate" /> |
```
Comment (RoRoJ) [inline storage/object/api-cli/using-api-call-list.mdx:50]:
```suggestion
|  [GET bucket website](/storage/object/api-cli/bucket-operations/#get-bucket-website) |  Returns the bucket Website basic configuration | <Icon name="validate" /> |
```

File: storage/object/api-cli/using-api-call-list.mdx [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+|  [POST object](/storage/object/api-cli/object-operations/#post-object) |  Adds an object to a bucket by using HTML forms | <Icon name="validate" /> |
+| [POST object restore](/storage/object/api-cli/object-operations/#restore-object) |  Restores an object from Glacier | <Icon name="validate" /> |
+|  [PUT object](/storage/object/api-cli/object-operations/#put-object) |  Adds an object to a bucket | <Icon name="validate" /> |
+|  [PUT object copy](/storage/object/api-cli/object-operations/#put-object-copy) |  Copies an object | <Icon name="validate" /> |
+|  [PUT object ACL](/storage/object/api-cli/object-operations/#put-object-acl) |  Adds an ACL to an object | <Icon name="validate" /> |
+|  [PUT object tagging](/storage/object/api-cli/object-operations/#put-object-tagging) |  Adds one or several tags to an object | <Icon name="validate" /> |
+|  [Multipart](/storage/object/api-cli/object-operations/#multipart-uploads) |  Initiates, aborts, completes, lists, uploads, uploads copy | <Icon name="validate" /> |
```
Comment (RoRoJ) [inline storage/object/api-cli/using-api-call-list.mdx:83]: This description is kind of confusing

File: storage/object/api-cli/object-operations.mdx [hunk 11]
```diff
@@ -27,7 +27,7 @@ If the operation is successful no output will be returned.
 </Message>
 
 **Sample request**
-```sh no-copy
+```http
```
Comment (bene2k1) [inline storage/object/api-cli/object-operations.mdx:30]: I don't think ` ```http` is required here- it will not change anything

**Review by angristan — APPROVED** [review]

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**